Podcast/FetchUserSubscriptionsEndpointRequest.swift

```diff
@@ -25,7 +25,9 @@ class FetchUserSubscriptionsEndpointRequest: EndpointRequest {
     }
     
     override func processResponseJSON(_ json: JSON) {
-        let series = json["data"]["subscriptions"].map{ jsons in Series(json: jsons.1["series"]) }
+        let series = json["data"]["subscriptions"].map{ jsons in
+            Cache.sharedInstance.update(seriesJson: jsons.1["series"])
+        }
         series.forEach { s in s.isSubscribed = true }
```

Comment:
why don't we do 
`
let s = Cache.sharedInstance.update(seriesJson: jsons.1["series"]); s.isSubscribed = true; return s
`
in the map instead of another forEach loop

Reply:
Yeah could do that, didn't see that forEach there.